spec/requests/api/v1/digital_objects_spec.rb

```diff
@@ -137,30 +137,48 @@
         sign_in_project_contributor to: [:delete_objects, :read_objects], project: authorized_project
         allow(publication_adapter).to receive(:unpublish).and_call_original
         allow(publication_adapter).to receive(:update_doi).and_call_original
+        allow(digital_object_search_adapter).to receive(:index).with(instance_of(DigitalObject::TestSubclass)).and_return([true, []])
       end
 
-      it 'returns 204' do
+      it 'returns 200' do
         allow(publication_adapter).to receive(:unpublish_impl).with(authorized_publish_target, digital_object_matcher).and_return([true, []])
-        allow(digital_object_search_adapter).to receive(:remove).with(digital_object_matcher).and_return([true, []])
         delete "/api/v1/digital_objects/#{authorized_object.uid}"
```

Comment:
Maybe I am forgetting where we left things but I thought HTTP DELETE was supposed to really destroy it?

Reply:
I'm was going for minimal changes here so we didn't have to update UI js, and also this REST endpoint will go away soon in favor of the graphql endpoint.  Okay with keeping as is?  Or would you prefer that I change this and the UI code?

Reply:
if the REST endpoint is going to be removed, I withdraw my question.